### DISCUSSION TITLE
Token Management and Editing

### DISCUSSION THREAD

**Discussion by lorddarq** [webstudio-is/webstudio-community#178] [Wishlist]
Please make token management and editing separate from the way it is done right now. It's incredibly counterintuitive/frustrating because you cannot see them at a glance, nor can you just edit the values without having to know which element needs to be edited.

It would make sense to have a new menu entry on the left side, below the marketplace, where you set JSONs, variables, and modify or delete tokens. as is, it's not usable at all. just confusing.

**Comment by kof** [comment]
We agree https://github.com/webstudio-is/webstudio/issues/3901 pls follow this issue and the roadmap https://wstd.us/roadmap
